src/foundry/client/pixi/core/containers/sprite-mesh.d.ts

```diff
@@ -0,0 +1,162 @@
+export {};
+
+declare global {
+  /**
+   * An extension of PIXI.Mesh which emulate a PIXI.Sprite with a specific shader.
+   */
+  // @ts-expect-error Foundry incorrectly overrode a private property
```

Comment:
What's the property? And why does this have to go on the class as a whole?

Reply:
Adjusted comment to note property. It goes on the class because that's where TS is flagging the error.